guizero/Painter.py

```diff
@@ -0,0 +1,26 @@
+from tkinter import Canvas, ALL
+from . import utilities as utils
+
+
+class Painter:
```

Comment:
The painter class should inherit the mixins it implements (Container, Widget, etc) and the tkmixins Canvas suports (Enable, Display, Schedule, etc).